server/seed/index.js

```diff
@@ -1 +1,18 @@
-// 
\ No newline at end of file
+const topics = require('./topics.js');
+const Topic = require('./../models/index.js').Topic;
+
+
+Topic.remove({}, function (err) {
```

Comment:
girl, I know you're not using callbacks!
can you turn this whole function into a regular promise chain with a catch at the end for errors?